tests/unit/acl.tcl

```diff
@@ -261,3 +261,42 @@ start_server {tags {"acl"}} {
         assert_match "*Unknown subcommand or wrong number of arguments*" $e
     }
 }
+
+set server_path [tmpdir "server.acl"]
+exec cp -f tests/assets/user.acl $server_path
+start_server [list overrides [list "dir" $server_path "aclfile" "user.acl"]] {
+    # user alice on allcommands allkeys >alice
+    # user bob on -@all +@set +acl ~set* >bob
+
+    test "Alice: can excute all command" {
+        r AUTH alice alice
+        assert_equal "alice" [r acl whoami]
+        r SET key value
+    }
+
+    test "Bob: just excute @set and acl command" {
+        r AUTH bob bob
+        assert_equal "bob" [r acl whoami]
+        assert_equal "3" [r sadd set 1 2 3]
+        catch {r SET key value} e
+        set e
+    } {*NOPERM*}
+
+    test "ACL load and save" {
+        r ACL setuser eve +get allkeys >eve on
+        assert_equal "OK" [r ACL save]
```

Comment:
these assertions are not needed.. if the command errors, the test will fail anyway (throws an exception).
you need this only if you're expecting different success responses (e.g. a number of an array)

Reply:
Thanks, already updated.